controllers/user/login.js

```diff
@@ -29,6 +34,15 @@ function renderForm(req, res, formValues = null, errors = []) {
     });
 }
 
+function renderRateLimitError(req, res) {
```

Comment:
Could we add a log when we render this: `logger.info('User rate limited')`

Reply:
Doing this in the main ratelimiter module already: https://github.com/biglotteryfund/blf-alpha/pull/2093/files#diff-f3fc288629418f5e498c36b048899b97R80

Reply:
Ah nice  👍